### PR TITLE
Apply V3 Design / Fieldset Wrapper (issue/v3-2129)

### PR DESCRIPTION
**Pull request by o-hook**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://github.com/porscheofficial/oss-docs/blob/67c5b24838a293ce7a964884e6005eb71f2b8579/CONTRIBUTOR_LICENSE_AGREEMENT.md)

#### References

- Preview: https://designsystem.porsche.com/issue/v3-2129/components/fieldset/examples

#### Scope

Short description of the pull request scope

### CODE REVIEW

File: packages/components-js/src/pages/fieldset-wrapper.html [hunk 1]
```diff
@@ -4,19 +4,31 @@
   }
 </style>
 
-<div class="playground light" title="should render with label">
+<div class="playground light" title="should render with label on light background">
```
Comment (denyo) [inline packages/components-js/src/pages/fieldset-wrapper.html:7]: Might be nice to apply the same grid similar as text-field-wrapper, textarea-wrapper and select-wrapper do it for a better overview on larger viewports.

File: packages/components/src/components/fieldset-wrapper/fieldset-wrapper-styles.ts [hunk 2]
```diff
@@ -2,18 +2,25 @@ import type { Theme } from '../../types';
...
+  state: FormState,
+  labelSize: FieldsetWrapperLabelSize,
+  hasLabel: boolean,
+  theme: Theme
+): string => {
   return getCss({
     '@global': {
       ':host': {
-        display: addImportantToRule('block'),
+        ...addImportantToEachRule({
```
Comment (denyo) [inline packages/components/src/components/fieldset-wrapper/fieldset-wrapper-styles.ts:20]: Unneeded object composition.

File: packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ If the `p-fieldset-wrapper` is set to `required="true"`, only the label of the `
 is removed from all wrapped child components, as long as they are Porsche Design System form elements. You should still
 set required on the input of the wrapped form elements to ensure accessibility, and the support of screen readers.
 
-<Playground :markup="requiredMarkup"></Playground>
+<Playground :markup="requiredMarkup" :config="config"></Playground>
```
Comment (denyo) [inline packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md:39]: Odd that the config was applied on some playgrounds only.

File: packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md [hunk 4]
```diff
@@ -133,6 +133,10 @@ export default class Code extends Vue {
 
 <style scoped lang="scss">
   .state-markup > * {
-    margin-top: .5rem;
+    margin-top: 16px;
```
Comment (denyo) [inline packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md:136]: Could use spacing scss variable :)

File: packages/components/src/components/fieldset-wrapper/fieldset-wrapper.tsx [hunk 5]
```diff
@@ -8,13 +8,15 @@ import { Required } from '../common/required/required';
 import { FORM_STATES } from '../../utils';
 import type { FormState } from '../../utils/form/form-state';
 import { FIELDSET_WRAPPER_LABEL_SIZES } from './fieldset-wrapper-utils';
+import { Theme } from '../../types';
```
Comment (denyo) [inline packages/components/src/components/fieldset-wrapper/fieldset-wrapper.tsx:11]: Combine with imports in L3.

File: packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md (hunk 3 above)
Comment (o-hook) [reply packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md:39]: Sorry, what do you mean exactly? `:config` was added to all of the 6 playgrounds. Or do I miss something?

File: packages/components-js/src/pages/fieldset-wrapper.html (hunk 1 above)
Comment (o-hook) [reply packages/components-js/src/pages/fieldset-wrapper.html:7]: Done

File: packages/components-js/src/pages/fieldset-wrapper.html [hunk 6]
```diff
@@ -84,7 +152,19 @@
...
-<div class="playground light" title="should render with success state and success message">
+<div class="playground dark" title="should render with error state and slotted error message on dark background">
+  <p-fieldset-wrapper label="Some label" theme="dark" state="error">
+    <p-radio-button-wrapper state="error" label="Some label" theme="dark">
+      <input type="radio" name="some-name-1" />
+    </p-radio-button-wrapper>
+    <p-radio-button-wrapper state="error" label="Some label" theme="dark">
+      <input type="radio" name="some-name-1" />
+    </p-radio-button-wrapper>
+    <span slot="message">Some slotted error Message</span>
```
[the remark this answers is not in the document]
Comment (o-hook) [reply packages/components-js/src/pages/fieldset-wrapper.html]: Done

File: packages/components/src/components/fieldset-wrapper/fieldset-wrapper-styles.ts (hunk 2 above)
Comment (o-hook) [reply packages/components/src/components/fieldset-wrapper/fieldset-wrapper-styles.ts]: Done

File: packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md (hunk 4 above)
Comment (o-hook) [reply packages/components/src/components/fieldset-wrapper/fieldset-wrapper.examples.md]: Done

File: packages/components/src/components/fieldset-wrapper/fieldset-wrapper.tsx (hunk 5 above)
Comment (o-hook) [reply packages/components/src/components/fieldset-wrapper/fieldset-wrapper.tsx]: Done

**Review by denyo — APPROVED** [review]

**Pull request merged**